src/modules/land_detector/MulticopterLandDetector.cpp

```diff
@@ -34,6 +34,28 @@
 /**
  * @file MulticopterLandDetector.cpp
  *
+ *The MC land-detector goes through 3 states before it will detect landed:
+ *
+ *State 1 (=ground_contact):
+ *ground_contact is detected once the vehicle is not moving along the NED-z direction and has
+ *a thrust value below 0.3 of the thrust_range (thrust_max - thrust_min). The condition has to be true
```

Comment:
don't you want to relate to hover thrust? might be still vague if the parameter is not set correctly but a relevant check if it is actually set to the right value. Could still default to the exact same check if all the parameters are left default...